### PR TITLE
Add best rated tracks and albums

### LINKED ISSUE #134: Show artist's best rated tracks and albums
**Is your feature request related to a problem? Please describe.**

Add to the artist page a list of the best-rated albums and tracks if it exists.

**Describe the solution you'd like**

On issue #132 will be added to the artist request the fields needed to get the best-rated albums and tracks of the artist, you can add a list with the same design as the top tracks and albums list that is already on the artist page.

The best-rated albums and tracks must be shown only if there are any rated albums and tracks.

**Additional context**
Depends on #132

### PR DESCRIPTION
**Pull request by lucasaarcoverde**
Closes #134 

Captura de Tela 2021-03-22 às 09 03 34

### CODE REVIEW

File: musicritic/src/components/artist/ArtistPageContent.css [hunk 1]
```diff
@@ -1,48 +1,48 @@
...
+    font-size: 3rem;
+    line-height: 4rem;
 }
 
 .discography-section-title {
-  font-size: 2rem;
-  line-height: 3rem;
-  text-transform: uppercase;
+    font-size: 2rem;
+    line-height: 3rem;
```
Comment (lucasaarcoverde) [inline musicritic/src/components/artist/ArtistPageContent.css:22]: This is the only real change

File: server/src/artist/artistController.js [hunk 2]
```diff
@@ -45,28 +49,45 @@ router.get('/artists/:id', async (req, res) => {
...
-    const bestRatedAlbums = albumsRatings.map(
-        ratings => [ratings[0], averageRating(ratings[1])]
-    ).sort((a, b) => b[1] - a[1]).slice(0, 5)
+    const bestRatedAlbums = albumsRatings
+        .map(ratings => [ratings[0], averageRating(ratings[1])])
+        .sort((a, b) => b[1] - a[1])
+        .slice(0, 5)
+        .map(([album, average]) => {
+            return { ...album, average };
+        });
```
Comment (lucasaarcoverde) [inline server/src/artist/artistController.js:65]: same

File: server/src/artist/artistController.js [hunk 3]
```diff
@@ -45,28 +49,45 @@ router.get('/artists/:id', async (req, res) => {
...
+    const tracksAverages = tracksRatings.map(ratings => [
+        ratings[0],
+        averageRating(ratings[1]),
+    ]);
+    const bestRatedTracks = tracksAverages
+        .sort((a, b) => b[1] - a[1])
+        .slice(0, 5)
+        .map(([track, average]) => {
+            return { ...track, average };
+        });
```
Comment (lucasaarcoverde) [inline server/src/artist/artistController.js:83]: same

**Review by JRobsonJr — APPROVED** [review]
:sparkles:

**Pull request merged**